### PR TITLE
add travis yml

### PR DESCRIPTION
**Pull request by stegben**

### CODE REVIEW

File: travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+  - "6"
+  - "4"
+  - "node"
```
Comment (chentsulin) [inline travis.yml]: What is it?

Comment (stegben) [inline travis.yml]: the latest stable version of node can be shorthanded like this

Comment (chentsulin) [inline travis.yml]: I usually use something like this: 
- "stable"

**Comment by stegben** [comment]
done

**Pull request merged**